client/src/main/scala/org/apache/celeborn/client/ApplicationHeartbeater.scala

```diff
@@ -96,8 +99,31 @@ class ApplicationHeartbeater(
     }
   }
 
+  private def unregisterApplication(): Unit = {
+    try {
+      // Then unregister Application
+      val response = masterClient.askSync[ApplicationLostResponse](
+        ApplicationLost(appId),
+        classOf[ApplicationLostResponse])
+      logInfo(s"Unregister Application $appId with response status: ${response.status}")
+    } catch {
+      case e: Exception =>
+        logWarning("AskSync unRegisterApplication failed.", e)
```

Comment:
We should add appId to the log

Reply:
> We should add appId to the log

Actually it works in driver, for we actually know what the application is~.